packages/transaction-controller/src/utils/gas.ts

```diff
@@ -141,6 +141,11 @@ async function getGas(
     return [txMeta.txParams.gas, undefined, txMeta.txParams.gas];
   }
 
+  if (txMeta.txParams.gasLimit) {
```

Comment:
Rather than adding the support here specifically, is now a good time to instead remove all usages of `gasLimit` and mark it as deprecated in the type.

Plus update the `NORMALIZERS` in `utils.ts` to set `gas` to `gasLimit` if `gas` is `undefined`?  

I'm assuming `gas` is the best choice going forward as it's the only one defined in our external doc and on `ethereum.org`.